.travis.yml

```diff
@@ -3,6 +3,14 @@ go_import_path: github.com/ethereum/go-ethereum
 sudo: false
 matrix:
   include:
+  - if: commit_message =~ ci:test-bootnode-availability
```

Comment:
TBH I don't see a reason to run this on CI.

Reply:
My reasoning for it:  If it's a "test" (or heuristic-generating tool) is something we're ever going to use as developers manually, we should do what we can to make the it (and comparable results) public.

> This heuristic just gets a little better when it's at least publicly available (cite-able, requires less/different trust, reproduceable).